src/user/user.service.ts

```diff
@@ -67,12 +72,19 @@ export class UserService {
     })
   }
 
-  async findAllUsers(): Promise<User[]> {
-    return await this.userDatabaseService.findAllUsers()
+  async getAllUsers(): Promise<User[]> {
+    return await this.userDatabaseService.getAllUsers()
   }
 
-  async findUserByUsername(username: string): Promise<User> {
-    return await this.userDatabaseService.findUserByUsername(username)
+  async getUserById(id: string): Promise<User> {
+    return await this.userDatabaseService.getUserById(id)
+  }
+
+  async getUserTodos(userId: string): Promise<Todo[]> {
+    await this.getUserById(userId)
+
+    // ? Question: can I skip this TodosService step and call TodosDatabaseService method directly?
```

Comment:
short answer is “yes”, but need to think twice which of the approaches will give you clearer architecture and responsibility chain